template/App.tsx

```diff
@@ -56,43 +56,56 @@ function Section({children, title}: SectionProps): React.JSX.Element {
 }
 
 function App(): React.JSX.Element {
+  return (
+    <SafeAreaProvider>
+      <MainContent />
+    </SafeAreaProvider>
+  );
+}
+
+function MainContent(): React.JSX.Element {
+  const insets = useSafeAreaInsets();
+
   const isDarkMode = useColorScheme() === 'dark';
 
   const backgroundStyle = {
     backgroundColor: isDarkMode ? Colors.darker : Colors.lighter,
+    bottomPadding: insets.bottom,
   };
 
+  const contentContainerStyle = { paddingBottom: insets.bottom };
+
   return (
-    <SafeAreaView style={backgroundStyle}>
-      <StatusBar
-        barStyle={isDarkMode ? 'light-content' : 'dark-content'}
-        backgroundColor={backgroundStyle.backgroundColor}
-      />
-      <ScrollView
-        contentInsetAdjustmentBehavior="automatic"
```

Comment:
replace  iOS only `contentInsetAdjustmentBehavior` with code that can work on both iOS and Android